package.json

```diff
@@ -45,7 +45,6 @@
     "jest": "jest",
     "jest:coverage": "jest --coverage",
     "pretest": "npm run lint",
-    "preinstall": "npm run clean:all",
```

Comment:
> I've also hooked into the npm install lifecycle (using preinstall lifecycle hook). When a user runs `npm install`, the script will clean all the `node_modules` directories, then npm will install new modules into node_modules directories.

I missed that this leaves an empty `node_modules` directory in the root directory and that only node_modules in the individual terra-component package directories remain. 

This is due to [an issue](npm/npm#10379) with how `preinstall` works since npm v3. scripts defined in `preinstall` run after node_modules are installed. So what actually happens when running `npm install` in the root directory is:
* node_modules installed in root
* (preinstall scripts run) clean:all runs and removes root node_modules and individual package's node_modules
* (postinstall script runs) lerna bootstraps packages by npm linking them and running npm install for each package.

Unfortunately, it doesn't look like the npm team will revert `preinstall` behavior to match how it worked with npm v2. So its better to drop our clean task from being called on `preinstall`. I do think it would be nice to set up a task that could be run from root directory that does the following in order:
* npm run clean:all
* npm install
* npm run bootstrap